app/models/concerns/publishing_api/content.rb

```diff
@@ -62,17 +62,41 @@ def content
       values_from_json_paths = INDEXABLE_CONTENT_VALUES_JSON_PATHS.map do |item|
         item.on(document_hash).map { |body| BodyContent.new(body).html_content }
       end
+
       values_from_parts = document_hash.dig(:details, :parts)&.map do |part|
         # Add the part title as a heading to help the search model better understand the structure
         # of the content
         ["<h1>#{part[:title]}</h1>", BodyContent.new(part[:body]).html_content]
       end
 
-      [*values_from_json_paths, *values_from_parts]
+      [*values_from_json_paths, *values_from_parts, *values_from_blocks(document_hash)]
         .flatten
         .compact_blank
         .join(INDEXABLE_CONTENT_SEPARATOR)
         .truncate_bytes(INDEXABLE_CONTENT_MAX_BYTE_SIZE)
     end
+
+    def values_from_blocks(document_hash)
```

Comment:
I don't think `document_hash` needs to be passed into this method. The concern should already have access to it.